android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageKeystoreAesCbc.java

```diff
@@ -159,6 +161,9 @@ public EncryptionResult encrypt(@NonNull final DecryptionResultHandler handler,
           new EncryptionContext(safeAlias, key, username, password);
 
         handler.askAccessPermissions(context);
+        if (handler.getError() != null && handler.getError().getMessage().contains("code: " + BiometricPrompt.ERROR_NEGATIVE_BUTTON)) {
+          throw new CryptoFailedException(handler.getError().getMessage());
```

Comment:
Could we extract `handler.getError()` to `Error handlerError = handler.getError()`
```suggestion
       Error handlerError = handler.getError()
        if (handlerError != null && handlerError.getMessage().contains("code: " + BiometricPrompt.ERROR_NEGATIVE_BUTTON)) {
          throw new CryptoFailedException(handlerError.getMessage());
```